### DISCUSSION TITLE
Set hostname for datadog output

### DISCUSSION THREAD

**Discussion by axelzedigh** [fluent/fluent-bit#5507] [Q&A]
I want to use fluent-bit to tail log-files and to use datadog as the output. I use the `dd_tags` option to set tags for the log, e.g. `host:TEST-SERVER-1,os:linux` but when inspecting the logs in Datadog the host-tag has been lower-cased (`test-server-1`). Datadog is thus not able to bind the logs with other metrics associated with `host:TEST-SERVER-1`. Can the hostname be configured explicitly in fluent-bit? Any other work-around?

Edit:
This is solved by appending a query to the api-key, e.g:
```
[OUTPUT]
    Name                datadog
    Match               *
    Host                http-intake.logs.datadoghq.eu
    TLS                 on
    compress            gzip
    apikey              <API-KEY>?hostname=TEST-SERVER-1
    dd_service          test
    dd_source           python
    dd_tags             market:europe
    dd_message_key      log
```

**Comment by mumumumu** [comment]
This workaround doesn't seem to work anymore. I'm guessing due to the plugin being updated to the v2 Datadog API in June 2022.

The [v2 API](https://docs.datadoghq.com/api/latest/logs/) does support passing a `hostname` param in the request body (along side the `service`, `ddsource`, and `ddtags` that are already being passed today).

Seems the Datadog output plugin would need to be updated to support something like a `dd_hostname` config param.
